### PR TITLE
Identifies cache invalidation problem, reduces ttl as a mitigation

### PR DESCRIPTION
**Pull request by leogrim**
@stkem @andrewconner @eishay @raymondng

### CODE REVIEW

File: kifi-backend/modules/shoebox/app/com/keepit/model/SocialConnectionRepo.scala [hunk 1]
```diff
@@ -64,10 +64,11 @@ object SocialConnectionInfo {
 
 case class SocialUserConnectionsKey(id: Id[SocialUserInfo]) extends Key[Seq[SocialConnectionInfo]] {
   val namespace = "social_user_connections"
-  override val version = 2
+  override val version = 3
```
Comment (raymondng) [inline kifi-backend/modules/shoebox/app/com/keepit/model/SocialConnectionRepo.scala]: Just curious: is this necessary?

**Comment by raymondng** [comment]
lgtm

File: kifi-backend/modules/shoebox/app/com/keepit/model/SocialConnectionRepo.scala (hunk 1 above)
Comment (leogrim) [inline kifi-backend/modules/shoebox/app/com/keepit/model/SocialConnectionRepo.scala]: No, just meant to see if that actually fixed my problem, but you're right, I can just clear the one problematic key manually.

**Comment by leogrim** [comment]
See lazy invalidation trick in UserCommander

**Pull request merged**